packages/react/src/errors/errors.ts

```diff
@@ -0,0 +1,7 @@
+export {
+  isClerkAPIResponseError,
+  isEmailLinkError,
+  isKnownError,
+  isMetamaskError,
+  EmailLinkErrorCode,
```

Comment:
Do we need both isEmailLinkError and EmailLinkErrorCode?

Reply:
How about exporting ClerkRuntimeError and ClerkAPIResponseError?

Reply:
>Do we need both isEmailLinkError and EmailLinkErrorCode?
I think we should keep only the `isEmailLinkError` 
> How about exporting ClerkRuntimeError and ClerkAPIResponseError?
I will :)